modules/extensions-xml-support/src/test/java/org/mule/test/functional/ModuleWithImplicitConfigurationUsingScopesTestCase.java

```diff
@@ -13,9 +13,11 @@
 import org.mule.test.petstore.extension.PetStoreClient;
 
 import io.qameta.allure.Feature;
+import org.junit.Ignore;
 import org.junit.Test;
 
 @Feature(XML_SDK)
+@Ignore("W-12121045: Ignore until we can set MULE_DISABLE_XML_SDK_IMPLICIT_CONFIGURATION_CREATION to false")
```

Comment:
Shouldn't this tests set the property to false, so that they test the behavior when the new feature is enabled?

Reply:
The problem is that the extension can be compiled